doc/getting_started.md

```diff
@@ -0,0 +1,36 @@
+# Getting started
+Now that you have successfully installed Azure CLI 2.0 and added the Azure DevOps extension, you are all set to get started!
+
+## Log in to Azure DevOps
+Before you can work with Azure DevOps, you need to log in to Azure CLI using the `az login` command.
+
+If the CLI can open your default browser, it will do so and load a sign-in page.

```

Comment:
I will recommend that we tell that CLI supports
1. 'az login'
2. 'az devops login'
and ask user to use -h in order to explore/ use the commands because
1. This instruction right now only looks incomplete (like --allow-no-subscriptions is missing)
2. This instruction will get outdate as and when these command are updated

Reply:
For 1&2 - The getting started page is all about getting set up in the major use case path, which I see to be az login. The only use case for az devops login is the PAT which has been included as a link out now. In case it changes, we will have to update documentation accordingly. 

2. Will update the documentation to include usage reference of -h to explore commands.

3. Can you elaborate on this further? Is allow no subscription the only missing doc? Further, going back to point 1&2, this is the getting started documentation and I would potentially call this another example scenario and move it as a separate page.

4. This is a constant problem which we have to take care of anyway irrespective of the data on getting started page.